cartridge-cli.lua

```diff
@@ -3427,12 +3498,40 @@ function cmd_pack.callback(args)
     end
 
     if app_state.tarantool_is_enterprise and app_state.build_in_docker then
-        if app_state.download_token == nil or app_state.download_token == '' then
-            die(
-                'Tarantool download token is required to pack enterprise Tarantool app in docker. ' ..
-                'Please, specify it using --download_token option or TARANTOOL_DOWNLOAD_TOKEN env variable'
-            )
+        local sdk_path, err = detect_sdk_path(args)
+        if sdk_path == nil then die(err) end
+
+        -- check that specified path is an existent directory
+        if not fio.path.exists(sdk_path) then
```

Comment:
You can omit this check. In case then directory doesn't exist second check 'is_dir' returns false

Reply:
I think that `Path doesn't exists` error is much better then `Path isn't a directory`.